### PR TITLE
Add more prose to the RTP integration section

### LINKED ISSUE #113: Expand RTP example appendix
In #96 / #111, we stripped out any dependencies on RTP, but we lost some of the details on per-packet vs. per-frame.  We should expand the RTP example appendix to discuss these cases.

### PR DESCRIPTION
**Pull request by bifurcation**
Fixes #113

### CODE REVIEW

**Review by suhasHere — APPROVED** [review]

File: draft-ietf-sframe-enc.md [hunk 1]
```diff
@@ -1143,13 +1153,35 @@ is only around 1%.
...
+receivers will need to be configured to known which of the above schemes has
+been applied to a given sequence of RTP packets. SFrame does not provide a
+mechanism for distributing this configuration information. In applications that
+use SDP for negotiating RTP media streams {{RFC4566}}, an appropriate extension
+to SDP could provide this function.
+
+Applying SFrame per-frame also requires that packetization and depacketization
+be done in a generic manner that does not depend on the media content of the
+packets, since the content being packetized / depacketized will be opaque
+ciphertext except for the SFrame header.  In order for such a generic
```
Comment (GhostofCookie) [inline draft-ietf-sframe-enc.md:1182]:
```suggestion
ciphertext (except for the SFrame header).  In order for such a generic
```

File: draft-ietf-sframe-enc.md [hunk 2]
```diff
@@ -179,42 +184,47 @@ complexity, at the cost of higher bandwidth consumption.  Some quantitative
...
-will depend on the environment in which SFrame is being integrated.
+applied in other scenarios.  The important thing is that the sender and
+receivers of an SFrame-encrypted object agree on that object's semantics.
+SFrame does not provide this agreement; it must be arranged by the application.
 
 ~~~ aasvg
       +--------------------------------------------------------+
       |                                                        |
       |  +----------+      +-------------+      +-----------+  |
  .-.  |  |          |      |             |      |   SRTP    |  |
```
Comment (GhostofCookie) [inline draft-ietf-sframe-enc.md:195]: Should we make this a bit more general than SRTP? Could this say HBH instead? Same for the Unprotect on line 220

File: draft-ietf-sframe-enc.md [hunk 3]
```diff
@@ -159,18 +158,24 @@ overhead only once per media frame, instead of once per packet.
...
-particular environment.
-
-For example, {{media-stack}} shows a typical media stack that takes media in
-from some source, encodes it into frames, divides those frames into media
-payloads, and then sends those payloads in SRTP packets.  Arrows indicate the
-points where SFrame protection would be integrated into this media stack, when
-applied per-frame or per-packet.
+SFrame is a general encryption framing, intended to be used as an E2E encryption
+layer over an underlying HBH-encrypted transport such as SRTP or QUIC
+{{RFC3711}}{{?I-D.ietf-moq-transport}}.
```
[the remark this answers is not in the document]
Comment (bifurcation) [reply draft-ietf-sframe-enc.md:163]: I think the "such as" captures it.

File: draft-ietf-sframe-enc.md [hunk 4]
```diff
@@ -179,42 +184,47 @@ complexity, at the cost of higher bandwidth consumption.  Some quantitative
 discussion of these trade-offs is provided in {{overhead-analysis}}.
 
 As noted above, however, SFrame is a general media encapsulation, and can be
-applied in other scenarios.  The precise efficiency and complexity trade-offs
-will depend on the environment in which SFrame is being integrated.
+applied in other scenarios.  The important thing is that the sender and
+receivers of an SFrame-encrypted object agree on that object's semantics.
+SFrame does not provide this agreement; it must be arranged by the application.
```
[the remark this answers is not in the document]
Comment (bifurcation) [reply draft-ietf-sframe-enc.md:189]: A little.  That section says:

> To use SFrame, an application needs to define the inputs to the SFrame
> encryption and decryption operations, and how SFrame ciphertexts are delivered
> from sender to receiver (including any fragmentation and reassembly).

I would propose we call that good enough for now, and give others a chance to opine on how we should evolve this section forward.

**Pull request merged**